inst/example_apps/rmd_dashboard_app/app.R

```diff
@@ -1,15 +1,22 @@
-
-library(shiny)
 library(crunchy)
-# Render the dashboard
-rmarkdown::render("dashboard.Rmd")
+
 ui <- fluidPage(
-    includeHTML("dashboard.html")
+     uiOutput("dashboard")
 )
 
-server <- function(input, output) {
+server <- function(input, output, session) {
+    user <- shinyUser()
+    output$dashboard <- renderUI({
+        current_user <- email(user())
+        if (grepl("acme.com$", current_user)) {
+            includeHTML("acme_dashboard.html")
+        } else if (grepl("globex.com$", current_user)) {
+            includeHTML("globex_dashboard.html")
+        } else {
+            h1("You are not logged in, please visit app.crunch.io to login")
```

Comment:
Not quite true; if you weren't logged in, L10 would have raised an error and you wouldn't have reached this line. This is just for authenticated users that don't match those email domains.